db/controllers/api.controller.js

```diff
@@ -53,3 +54,30 @@ exports.getComments = (req, res) => {
       res.status(500).send({ msg: "Internal Server Error" });
     });
 };
+
+exports.postComment = (req, res) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
+    return res.status(400).send({ msg: "Bad request" });
+  }
+
+  selectArticleById(article_id)
```

Comment:
This is not required as PSQL will throw an error as this id is a foreign key for this insertion. As such you should handle this with a psql error handler.